mainwindow.ui

```diff
@@ -1,24 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Not really sure what happened in this file but it's out of scope for this diff, can you revert it plz?